.github/workflows/e2e.yml

```diff
@@ -11,13 +11,15 @@ jobs:
     strategy:
       matrix:
         helm-version:
-          - 3.5.3
+          - 3.6.0
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Setup Kubernetes
         uses: engineerd/setup-kind@v0.5.0
+        with:
```

Comment:
Here is an example https://github.com/fluxcd/flux2/blob/main/.github/workflows/e2e.yaml#L27-L30